source/administration/changelog.md

```diff
@@ -4,9 +4,269 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+  - `/open` command to switch or join a channel.
+  - `/search` command to search text in messages.
+  - `/settings` command to open the Account Settings dialog.
+- `/invite_people` slash command is now disabled when account creation is set to false.
+- If a message starts with a / but fails to send (either due to timeout or invalid command), the message is put back to the input box.
+
+#### Bulk Import Tool
+- Added support for direct message channels and posts to the [bulk import tool](https://docs.mattermost.com/deployment/bulk-loading.html).
+
+#### Authentication
+- User creation via OAuth (GitLab/Google/Office365) properly restricted to accepted domains, [if specified](https://docs.mattermost.com/administration/config-settings.html#restrict-account-creation-to-specified-email-domains).
+- **Invite New Member** dialog validates email address against accepted domains, if set.
+
+#### New URL Routes // XXX Need better name and description of benefit
+- Added the ability to direct message by email or username with the following new routes for direct message channels: 
+  - `.../teamname/messages/@username`
+  - `.../teamname/messages/email`
+  - `.../teamname/messages/user_id` (redirects to `...teamname/messages/@username`)
+  - `.../teamname/messages/id1_id2` (redirects to `...teamname/messages/@username`)
+- Also added a new route for group message channels: 
+  - `.../teamname/messages/generated_id`
+
+#### Link Previews
+- After posting a message containing an image link, a preview is loaded only if one is available.
+
+#### Enterprise Edition
+- When a SAML user uses a non-supported locale, the language now defaults to English preventing login issues.
+
+### Bug Fixes
+- Emoji picker now closes in Firefox when clicking outside of it.
+- [...] menu no longer disappears in the comment thread when hovering over another post.
+- New direct messages received while in no teams do not show as unread after rejoining a team.
+- Fixed JavaScript errors when receiving messages when not belonging to a team.
+- An empty push notification no longer sent for messages only containing file attachments.
+- Custom emoji search results no longer filter by creator's first and last name.
+- `/expand` and `/collapse` slash commands now properly collapse images in website link previews.
+- Group Message channels that are favorited can now be closed.
+- Deactivated users now properly listed in Direct and Group Message channels in the left-hand sidebar.
+- Fixed search in team and channel Manage Members dialog.
+- File upload cancelled if you click "x" on thumbnail while file is uploading in your message draft.
+- Status no longer appears offline after joining a new team.
+- An empty push notification is no longer sent for messages only containing file attachments.
+- If you're at the top of a channel when a new post is received, the view no longer shifts and removes the oldest post that was in your view.
```

Comment:
Not really sure what this means. Is it that the center channel maintains scroll postition when new messages are received in the channel?